aspnetcore/fundamentals/routing.md

```diff
@@ -2,65 +2,20 @@
 title: Routing in ASP.NET Core
 author: rick-anderson
 description: Discover how ASP.NET Core routing is responsible for mapping request URIs to endpoint selectors and dispatching incoming requests to endpoints.
+monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 06/17/2019
+ms.date: 09/16/2019
 uid: fundamentals/routing
 ---
 # Routing in ASP.NET Core
 
 By [Ryan Nowak](https://github.com/rynowak), [Steve Smith](https://ardalis.com/), and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-::: moniker range="<= aspnetcore-1.1"
```

Comment:
What's the gist of the moving around of content? It's hard for me to get a sense of what this doc is going to look like when someone reads the 3.0 version.

Reply:
It would be great if someone could give me an idea of what parts of this are new and need review.